doc/STYLEGUIDE.md

```diff
@@ -157,8 +160,49 @@ Put a blank line between each group of imports.  The imports in each
 group should be sorted alphabetically, by module name.
 
 Always use explicit import lists or `qualified` imports for standard
-and third party libraries. The *Prelude* is an exception.
+and third party libraries.  The *Juvix Standard Library* is an exception.
 
+Qualified imports help two fold:
```

Comment:
I thought we are putting the motivation in a footnote?

Reply:
Well there are no other footnotes in the document, and the document isn't expected to grow too much, so it'll be odd to add it as such here

Reply:
hmm...I prefer otherwise because people tend to not digest as well when there's a lot of info. I want people to follow the guide well. But it's up to you. No worries. :)

Reply:
Let's leave it here for now, but we can separate out footnotes if the guide continues to grow